vectorc.go

```diff
@@ -105,6 +105,9 @@ func RelativeGeo(src, dest Location3D) (distance3d, thetaH, thetaV float64) {
 	thetaV = cmplx.Phase(complex(r, z))
 	// thetaV = math.Acos(z / r)
 	distance3d = dest.DistanceFrom(src)
+	if thetaH < 0 {
```

Comment:
Thanks.. Fixed it .. could not merge, because the pull had older commits too. Hope this validated well.